common/qconnector/rabbitmq_connector.py

```
@@ -0,0 +1,125 @@
+        if self.__rmq_conn:
+            self.__rmq_conn.close()
+
+    def publish(self, routing_key, payload):
+        """Publish with connection/channel state verification and message preservation"""
+        compressed_body = compress(payload)
+        logger.info('sending message with routing_key %s compressed_body bytes %s ',
+                  routing_key, len(compressed_body))
+        
+        try:
+            self.basic_publish(routing_key, compressed_body)
+            logger.info(f"Message published successfully")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)